index.html

```diff
@@ -64,12 +84,12 @@
         </div>
         <div class="flex justify-center gap-[0.75rem] mb-[2rem]">
           <button
-            class="font-arial bg-secondary-white text-sm text-tertiary-gray rounded-lg hover:shadow-shadow1 border-solid border-primary-white pl-[1rem] p-[0.5rem] pb-[0.5rem] pr-[1rem]"
+            class="font-arial bg-secondary-white text-sm text-tertiary-gray rounded-lg hover:shadow-shadow1 border-solid border-primary-white pl-[1rem] p-[0.5rem] pb-[0.5rem] pr-[1rem] nav"
```

Comment:
aslo here remove the `nav` class , check well , I think  padding of 1 rem exists in tailwind, no need to make it custom